lib/Mojolicious/resources/templates/mojo/debug.html.ep

```diff
@@ -170,24 +177,36 @@
                   <script>
                     let current = '#context';
                     function mojoShowcase() {
-                      $('#showcase').on('click', function () {
-                        $(this).unbind('click');
-                        $(current).slideToggle('slow', function () {
-                          current = current == '#context' ? '#insight' : '#context';
-                          $(current).slideToggle('slow', function () {
-                            mojoShowcase();
-                          });
-                        });
-                      });
+                      const showcase = document.querySelector('#showcase');
+                      if (showcase === null) {
+                        console.error('#showcase is null');
```

Comment:
Why do we care about this?

Reply:
You're overthinking this. It will just be dead code.

Reply:
Said and done, fixing it.

Reply:
I don't want to see `console.*` calls unless they actually provide information that an exception would not. And not for things that can't really happen.

Reply:
Here for `#showcase` not to exist someone else would have to have really messed up with another patch. Preparing for that just doesn't make sense.

Reply:
Removed, sorry for the incovenience.